.travis.yml

```diff
@@ -320,7 +333,7 @@ script:
   - $CXX --version
 
   # compile and execute unit tests
-  - mkdir -p build && cd build
+  - mkdir -p unit-test-build && cd unit-test-build
```

Comment:
Why changing the name of the directory here?

Reply:
I think because on MacOS (the build which was failing) there was a case sensitivity naming conflict. The file "BUILD" was conflicting with "build". I don't think you can rename `BUILD` but I will look into it.

Reply:
I see.